config/prow/cluster/cerberus_configmaps.yaml

```diff
@@ -0,0 +1,656 @@
+            database_path = config["database"].get("database_path", "/tmp/cerberus.db")
+            reuse_database = config["database"].get("reuse_database", False)
+
+            # Initialize clients and set kube api request chunk size
+            if not os.path.isfile(kubeconfig_path):
+                kubeconfig_path = None
+            logging.info("Initializing client to talk to the Kubernetes cluster")
+            kubecli.initialize_clients(kubeconfig_path, request_chunk_size)
+
+            if "openshift-sdn" in watch_namespaces:
+                sdn_namespace = kubecli.check_sdn_namespace()
+                watch_namespaces = [namespace.replace('openshift-sdn', sdn_namespace)
+                                    for namespace in watch_namespaces]
+
+            # Check if all the namespaces under watch_namespaces are valid
+            #watch_namespaces = kubecli.check_namespaces(watch_namespaces)
+
+            # Cluster info
+            logging.info("Fetching cluster info")
+            if distribution == "openshift":
+                cluster_version = runcommand.invoke("kubectl get clusterversion")
+                logging.info("\n%s" % (cluster_version))
+            #cluster_info = runcommand.invoke("kubectl cluster-info | awk 'NR==1' | sed -r "
+            #                                 "'s/\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]//g'")  # noqa
+            cluster_info = "IKS Prow cluster"
+            logging.info("%s" % (cluster_info))
+
+            # Run http server using a separate thread if cerberus is asked
+            # to publish the status. It is served by the http server.
+            if cerberus_publish_status:
+                if not 0 <= port <= 65535:
+                    logging.info("Using port 8080 as %s isn't a valid port number" % (port))
+                    port = 8080
+                address = ("0.0.0.0", port)
+                server_address = address[0]
+                port = address[1]
+                logging.info("Publishing cerberus status at http://%s:%s"
+                             % (server_address, port))
+                server.start_server(address)
+
+            dbcli.set_db_path(database_path)
+            if not os.path.isfile(database_path) or not reuse_database:
+                dbcli.create_db()
+                dbcli.create_table()
+
+            # Create slack WebCleint when slack intergation has been enabled
+            if slack_integration:
+                slack_integration = slackcli.initialize_slack_client()
+
+            # Run inspection only when the distribution is openshift
+            if distribution == "openshift" and inspect_components:
+                logging.info("Detailed inspection of failed components has been enabled")
+                inspect.delete_inspect_directory()
+
+            # get list of all master nodes to verify scheduling
+            # master_nodes = kubecli.list_nodes("node-role.kubernetes.io/master")
```

Comment:
Commenting call to `list nodes `as this needs sa to have a _role to list nodes._